setup.py

```diff
@@ -53,7 +53,7 @@
     package_dir={"": "py"},
     include_package_data=True,
     packages=setuptools.find_packages(where="py"),
-    python_requires=">=3.8.0",
```

Comment:
technically https://github.com/braintrustdata/autoevals/pull/154 will be the official deprecation of 3.8 and 3.9.